processor/processor.go

```diff
@@ -1583,6 +1590,8 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 	// map of jobID to destinationID: for messages that needs to be delivered to a specific destinations only
 	jobIDToSpecificDestMapOnly := make(map[int64]string)
 
+	nonEventStreamSourceIds := make(map[string]bool)
```

Comment:
Let's compute this once in `configBackendSubscriber` and use it across the code? Any issues with this approach ?

Reply:
Its better to keep it in memory instead of calculating again and again. Assuming that data won't be too much to keep in memory.